integration/getClientConfigTestData/no-client-id.golden

```diff
@@ -2,7 +2,9 @@ Usage:
   mobile get clientconfig <clientID> [flags]
 
 Flags:
-  -h, --help   help for clientconfig
+      --allow-invalid-certs   include certificate hashes for services with invalid/self-signed certificates
```

Comment:
as we have discussed in another PR, it might be better to call it `--allow-self-signed-certs`?